applications/migrations/0010_application_submitted_at_application_submitted_by_and_more.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 4.1.2 on 2022-12-13 14:22
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("applications", "0009_use_isnull_for_check_constraints"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="application",
+            name="submitted_at",
+            field=models.DateTimeField(null=True),
+        ),
+        migrations.AddField(
+            model_name="application",
+            name="submitted_by",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="submitted_applications",
+                to=settings.AUTH_USER_MODEL,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="application",
+            name="status",
+            field=models.TextField(
+                choices=[
+                    ("approved_fully", "Approved Fully"),
+                    ("approved_subject_to", "Approved Subject To"),
+                    ("completed", "Completed"),
+                    ("submitted", "Submitted"),
+                    ("ongoing", "Ongoing"),
+                    ("rejected", "Rejected"),
+                ],
+                default="ongoing",
+            ),
+        ),
+        migrations.AddConstraint(
+            model_name="application",
+            constraint=models.CheckConstraint(
+                check=models.Q(
+                    models.Q(
+                        ("submitted_at__isnull", True), ("submitted_by__isnull", True)
+                    ),
+                    models.Q(
+                        ("submitted_at__isnull", False), ("submitted_by__isnull", False)
+                    ),
+                    _connector="OR",
+                ),
+                name="applications_application_both_submitted_at_and_submitted_by_set",
+            ),
+        ),
+    ]
```

Comment:
Can you give this a more meaningful name? Maybe something like `added_application_submitted_fields`, it really helps when you're looking at the list of migrations and trying to work out what they do.

Reply:
Done. Although, we should probably look at squashing them fairly soon.